shellcheckrc

```diff
@@ -0,0 +1,8 @@
+# Allow opening any 'source'd file, even if not specified as input
+external-sources=true
+
+# Turn on warnings for unquoted variables with safe values
+enable=quote-safe-variables
+
+# Turn on warnings for unassigned uppercase variables
+enable=check-unassigned-uppercase
```

Comment:
We have a ton of UPPERCASE vars that are passed through several scripts and can be set or not. Just make sure we have proper default values for everyone of them in each script.